chat.quadrix.Quadrix.yml

```diff
@@ -0,0 +1,54 @@
+app-id: chat.quadrix.Quadrix
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos
+modules:
+  - name: quadrix
+    buildsystem: simple
+    skip-arches:
+      - aarch64
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://quadrix.chat/releases/deb/amd64/quadrix_0.6.5_amd64.deb
```

Comment:
so what about aarch64 users?

Reply:
OK will remove `skip-arches` and build a _aarch64_ .deb this weekend :-)

Reply:
But how can I skip the _aarch64_ test build until I have a _aarch64_ .deb file? The `skip-arches` option didn't seem to do anything.

Reply:
https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64